### PR TITLE
Feature/SK-486 | Implement kubernetes event listener

### PR DESCRIPTION
**Pull request by niklastheman**
- Event listener username and password (secret) added
- Added event listener deployment

### CODE REVIEW

File: scaleout/stackn/templates/event-listener-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      labels:
+        name: studio-event-listener
+        app: stackn-studio
+        allow-api-access: "true"
+    spec:
+      automountServiceAccountToken: true
+      serviceAccountName: studio
+      containers:
+      - image: {{ .Values.eventListener.image.repository }}
+        imagePullPolicy: Always
```
Comment (Wrede) [inline scaleout/stackn/templates/event-listener-deployment.yaml:25]: change to IfNotPresent

File: scaleout/stackn/templates/event-listener-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  template:
+    metadata:
+      labels:
+        name: studio-event-listener
+        app: stackn-studio
+        allow-api-access: "true"
+    spec:
+      automountServiceAccountToken: true
+      serviceAccountName: studio
+      containers:
```
Comment (Wrede) [inline scaleout/stackn/templates/event-listener-deployment.yaml:23]: lack of security context, see studio and other deplyments, requires that that image is run as non-root

**Review by Wrede — CHANGES_REQUESTED** [review]
Looks really good! Lets iterate on improvements

File: scaleout/stackn/templates/event-listener-deployment.yaml [hunk 3]
```diff
@@ -22,7 +22,7 @@ spec:
       serviceAccountName: studio
       containers:
       - image: {{ .Values.eventListener.image.repository }}
```
Comment (Wrede) [inline scaleout/stackn/templates/event-listener-deployment.yaml:24]: for the main container add:
{{- if .Values.studio.securityContext.enabled }}
        securityContext:
          runAsUser: {{ .Values.studio.securityContext.runAsUser }}
          runAsGroup: {{ .Values.studio.securityContext.runAsGroup }}
          allowPrivilegeEscalation: {{ .Values.studio.securityContext.allowPrivilegeEscalation }}
          privileged: {{ .Values.studio.securityContext.privileged }}
          capabilities:
            drop:
              - all
        {{- end }}
        {{- if .Values.studio.readinessProbe.enabled }}

**Review by Wrede — CHANGES_REQUESTED** [review]

File: scaleout/stackn/templates/event-listener-deployment.yaml (hunk 3 above)
Comment (niklastheman) [reply scaleout/stackn/templates/event-listener-deployment.yaml:24]: Alright man, will do!

**Pull request merged**